### PR TITLE
"yay

### PR DESCRIPTION
**Pull request by eshimel**
"

### CODE REVIEW

File: challenge.md [hunk 1]
```diff
@@ -29,22 +31,23 @@ Use `ls` to show the contents of `quiz` - was `temp` deleted?
...
- 
+ <!-- Answer Starts Here -->Absolute because you are pointing the URL of the font file.
+
  <!-- Answer Ends Here -->
 
 3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
 
  <!-- Answer Starts Here -->
- 
+ Relative, as long as the image lives inside the same projects.
```
Comment (GA-MEB) [inline challenge.md:50]: Yep, that's right. In general, you should use Relative paths to link to resources within the same project, and use Absolute paths to link to resources outside of that project.

**Comment by GA-MEB** [comment]
0 of 3 on Navigating the Filesystem
(It appears that you didn't submit anything. Did you add your quiz directory to the commit?)

3 of 3 on Absolute and Relative Paths

Let me know if you have any questions!

**Comment by eshimel** [comment]
Hmmm...I've got a pull request listed for the assessment  #31 called edie's
answer .....  I'm not sure what I did wrong that it's not showing up.  I'll
check in with you in the morning.

Thanks!

Edie

**Comment by eshimel** [comment]
I went ahead and recommitted the assessment and submitted a new pull
request.  Still not sure how the submission got so messy.

**Pull request closed without merging**